### PR TITLE
Added the option to switch between the 3 different types - subs, views and watch time.

### PR DESCRIPTION
**Pull request by fr3fou**
Hey!

This is my first time using p5.js, so if you have any criticism / feedback, I'm all ears.

I basically added a select input above the canvas, a switch case which changes the type + color (feel free to change it) and renamed the `maxSubs` and `minSubs` variables to something more generic - `maxValue` and `minValue`, as they are now used for more stuff, not just subs.

```js
  let maxValue = 0; 
  let minValue = Infinity;
```

```js
  select = createSelect();
  select.option('Subscribers');
  select.option('Watch Time');
  select.option('Views');
  select.changed(processData);
```

```js
  let type = (() => {
    switch (select.value()) {
      case 'Views':
        currentColor = color(255, 0, 200, 100);
        return 'views';
      case 'Watch Time':
        currentColor = color(200, 0, 100, 100);
        return 'watch_time_minutes';
      case 'Subscribers':
        currentColor = color(64, 250, 200, 100);
        return 'subscribers';
    }
  })();
```
###### I don't know if the code above is bad practice, It just seemed better than having a normal switch case. 

Also, did I mess something up, or is the data between the different types **too** similar?
###### I really hope it wasn't my mistake
I "debugged" (console.logged heh) the values for my country and looked them up in the .csv file and they were indeed correct. 
Speaking of .csv files, you haven't pushed the subscriber.csv file!

### CODE REVIEW

File: sketch.js [hunk 1]
```diff
@@ -33,57 +37,75 @@ function setup() {
...
 
-  let maxSubs = 0;
-  let minSubs = Infinity;
+  createP('Select category!');
+  select = createSelect();
+  select.option('Subscribers');
+  select.option('Watch Time');
+  select.option('Views');
+  select.changed(processData);
+  createP(''); // look, I don't know how to use p5 properly, I used this to push the canvas a little down ;w;
```
Comment (shiffman) [inline sketch.js]: The best way to handle this is probably to make divs in `index.html` add a `style.css` and use `parent()` to place any generated DOM elements in the divs. The DOM elements could also be written into the HTML itself and accessed here with `select()`.

**Review by shiffman — COMMENTED** [review]
This is great! Let me know if you want to adjust the DOM stuff or if you prefer it gets cleaned up later separate from this pull request.

**Comment by shiffman** [comment]
Also note I added the `subscribers_geo.csv` file here https://github.com/CodingTrain/500k-Subscriber-Map/commit/5200d6c80c1db2d4a4d15e579e08fbab5928a7ed

**Comment by fr3fou** [comment]
oops, I sorta messed up with rebasing / merge conflicts so I ended up duplicating my commmits ;w;
anyway, I adjusted the DOM stuff, so that they are now in the HTML. Feel free to merge this, I guess 
¯\\\_(ツ)\_/¯

**Comment by shiffman** [comment]
Sure thing, thank you!

**Pull request merged**